server/views/layout-react.jade

```diff
@@ -11,5 +11,18 @@ html(lang='en')
     script!= state
     script.
       window.webpackManifest = !{JSON.stringify(chunkManifest || {})};
+    script.
+      if (typeof localStorage !== 'undefined') {
+        let fccTheme;
+        try {
+          fccTheme = JSON.parse(localStorage.getItem('fccTheme'));
+        }
+        catch(e) {
+          fccTheme = null;
+        }
+        if (fccTheme && fccTheme === 'night') {
+          document.body.classList.add('night');
+        }
+      }
```

Comment:
I'd like to see this wrapped in a IIFE to prevent polluting the global scope.

Also, since we are using a try/catch we can reduce the amount of code here by removing the check for localStorage initially.

We can also move the `classList.add` call within the try of the try/catch.